lib/capybara/rack_test/form.rb

```diff
@@ -88,6 +69,8 @@ def add_input_param(field, params)
 
       Capybara::RackTest::Node.new(driver, field).value.to_s
     when 'file'
+      return if value.blank?
```

Comment:
I would have to set up a test case with other browsers to be sure, but I think this kind of short circuit would need to be in place for other `<input>` fields with blank values.

I'm not sure if there is in practice a distinction between an empty string value and a `nil` value, but the code here already does `field['value'].to_s` above.